Update xaml-hot-reload.md (#10360)
MicrosoftDocs/visualstudio-docs: docs/xaml-tools/xaml-hot-reload.md

---
title: "XAML Hot Reload for WPF and UWP apps"

> description: Modify XAML code in running .NETMAUI, WinUI 3, Windows Presentation Foundation (WPF), or Universal Windows Platform (UWP) apps with edit and continue features in XAML Hot Reload.

Reviewer: .NETMAUI needs a space hehe

> description: Modify XAML code in running .NETMAUI, WinUI 3, Windows Presentation Foundation (WPF), or Universal Windows Platform (UWP) apps with edit and continue features in XAML Hot Reload.

Author: ```suggestion
description: Modify XAML code in running .NET MAUI, WinUI 3, Windows Presentation Foundation (WPF), or Universal Windows Platform (UWP) apps with edit and continue features in XAML Hot Reload.
```

ms.subservice: xaml-tools
monikerRange: ">=vs-2019"
---

> # What is XAML Hot Reload ? 

Reviewer: no space before ? ??

> # What is XAML Hot Reload ? 

Author: ```suggestion
# What is XAML Hot Reload? 
```

## Where to get XAML Hot Reload

> Visual Studio XAML Hot Reload is currently supported only when you're running an application in **Visual Studio** -or- **Blend for Visual Studio** (If supported by the framework) with the debugger attached (**F5** or **Start debugging**).

Reviewer 2: "If supported" => "if supported" (as it's part of the same sentence). But what cases aren't supported - what was the driver for adding this caveat?

Author: the Blend for Visual Studio..... is kinda odd... i wasn't sure how to word it... maybe remove Blend? basically i don't think blend supports some of the frameworks.... i assume?

> Visual Studio XAML Hot Reload is currently supported only when you're running an application in **Visual Studio** -or- **Blend for Visual Studio** (If supported by the framework) with the debugger attached (**F5** or **Start debugging**).

Author: ```suggestion
Visual Studio XAML Hot Reload is currently supported only when you're running an application in **Visual Studio** -or- **Blend for Visual Studio** (if supported by the framework) with the debugger attached (**F5** or **Start debugging**).
```
